### DISCUSSION TITLE
ApexEmailNotifications weren't included in SGD

### DISCUSSION THREAD

**Discussion by zdinozi** [scolladon/sfdx-git-delta#932] [Q&A] [answered]
Hi,

I updated ApexEmailNotifications metadata. Change is visible in Pull Request: 

![image](https://github.com/user-attachments/assets/7841ed3f-dc51-4129-a773-c1b6a98b9529)

But delta didn't included that metadata in Package.xml

![image](https://github.com/user-attachments/assets/65e0675b-50cd-4920-b721-1eceacaa4450)

Is this a bug or am I doing something wrong?

**Comment by scolladon** [comment, marked as answer]
Hi !

This metadata is not yet handled by the plugin.
I convert this discussion to feature request and I'll ship a version supporting this metadata soon.
